users/stewryan/index.md

```diff
@@ -0,0 +1,29 @@
+---
+title: Ryan-personal
+layout: defaul
+author: Ryan Stewart
+---
+# Ryan Stewart
+--------------
+
+I am a first year PhD student at Oregon State University working under Dr. Todd Palmer. I obtained my masters degree from Idaho State University (ISU) while working under Dr. Chad Pope.
+
+My masters work focused on helping create a benchmark model for the Experimental Breeder Reactor II (EBR-II) for the International Reactor Physics Experiment Evaluation Project ([IRPhEP](http://irphep.inl.gov/)). The IRPhEP handbook is utilized by the reactor physics community as a tool to aid in the validation of calculational techniques and data. The focus of my work was to determine the viability of homogenization to specific areas of the core in the hopes of reducing the size of the input file, and the time required to run a simulation. To ensure the process produced accurate results, a sensitivity analysis was performed on a driver assembly for both a heterogeneous and homogeneous model. A full copy can be found [here](./files/ryan_thesis.pdf).
+
+For summer 2017, I worked in conjunction with Idaho National Lab (INL) and ISU on developing and implementing Functional Expansion Tallies (FETs) in the neutronics code Shift for the Exascale Computing Project. The FETs are meant to create a seamless transition for data from neutronics to computational fluid dynamics codes to allow for fully coupled reactor physics problems to be solved. A short paper on th work can be found [here](./files/Report.pdf).
```

Comment:
Just a couple grammatical things here. 

"For **the** summer **of** 2017..."

"..short paper on **the** work can be found..."